pybsn/test/fixtures/cassettes/switch/get_switches.json

```diff
@@ -0,0 +1 @@
\",\n  \"shutdown\" : false,\n  \"software-description\" : \"ivs 0.5\",\n  \"suspended-reason\" : \"No fabric role configured\"\n}, {\n  \"connected\" : true,\n  \"connected-since\" : \"2015-05-20T00:49:45.420Z\",\n  \"dpid\" : \"00:00:00:00:00:03:00:05\",\n  \"fabric-connection-state\" : \"suspended\",\n  \"fabric-last-seen-time\" : \"2015-05-20T00:49:48.560Z\",\n  \"handshake-state\" : \"master-state\",\n  \"inet-address\" : {\n    \"inet-port\" : 54895,\n    \"ip\" : \"192.168.142.212\"\n  },\n  \"model-number-description\" : \"ivs 0.5\",\n  \"serial-number-description\" : \"\",\n  \"shutdown\" : false,\n  \"software-description\" : \"ivs 0.5\",\n  \"suspended-reason\" : \"No fabric role configured\"\n}, {\n  \"connected\" : true,\n  \"connected-since\" : \"2015-05-20T00:49:45.410Z\",\n  \"dpid\" : \"00:00:00:00:00:03:00:06\",\n  \"fabric-connection-state\" : \"suspended\",\n  \"fabric-last-seen-time\" : \"2015-05-20T00:49:48.560Z\",\n  \"handshake-state\" : \"master-state\",\n  \"inet-address\" : {\n    \"inet-port\" : 39374,\n    \"ip\" : \"192.168.142.212\"\n  },\n  \"model-number-description\" : \"ivs 0.5\",\n  \"serial-number-description\" : \"\",\n  \"shutdown\" : false,\n  \"software-description\" : \"ivs 0.5\",\n  \"suspended-reason\" : \"No fabric role configured\"\n}, {\n  \"connected\" : true,\n  \"connected-since\" : \"2015-05-20T00:49:45.534Z\",\n  \"dpid\" : \"00:00:00:00:00:03:00:07\",\n  \"fabric-connection-state\" : \"suspended\",\n  \"fabric-last-seen-time\" : \"2015-05-20T00:49:48.562Z\",\n  \"handshake-state\" : \"master-state\",\n  \"inet-address\" : {\n    \"inet-port\" : 39765,\n    \"ip\" : \"192.168.142.212\"\n  },\n  \"model-number-description\" : \"ivs 0.5\",\n  \"serial-number-description\" : \"\",\n  \"shutdown\" : false,\n  \"software-description\" : \"ivs 0.5\",\n  \"suspended-reason\" : \"No fabric role configured\"\n}, {\n  \"connected\" : true,\n  \"connected-since\" : \"2015-05-20T00:49:45.558Z\",\n  \"dpid\" : \"00:00:00:00:00:03:00:08\",\n  \"fabric-connection-state\" : \"suspended\",\n  \"fabric-last-seen-time\" : \"2015-05-20T00:49:48.561Z\",\n  \"handshake-state\" : \"master-state\",\n  \"inet-address\" : {\n    \"inet-port\" : 45732,\n    \"ip\" : \"192.168.142.212\"\n  },\n  \"model-number-description\" : \"ivs 0.5\",\n  \"serial-number-description\" : \"\",\n  \"shutdown\" : false,\n  \"software-description\" : \"ivs 0.5\",\n  \"suspended-reason\" : \"No fabric role configured\"\n} ]", "encoding": null}, "headers": {"transfer-encoding": ["chunked"], "accept-ranges": ["bytes"], "vary": ["Accept-Charset", "Accept-Encoding", "Accept-Language", "Accept"], "server": ["nginx/1.1.19"], "connection": ["keep-alive"], "date": ["Wed", "20 May 2015 01:45:53 GMT"], "content-type": ["application/json"]}, "status": {"message": "OK", "code": 200}, "url": "https://192.168.142.213:8443/api/v1/data/controller/applications/bcf/info/fabric/switch"}, "recorded_at": "2015-05-20T01:45:53"}], "recorded_with": "betamax/0.4.2"}
```

Comment:
will probably have to mock up the login request as well.

additionally need to filter sensitive data like the controller ip.